sandbox/apparmor/apparmor.go

```diff
@@ -366,6 +366,12 @@ func probeKernelFeatures() ([]string, error) {
 			}
 		}
 	}
+	if data, err := os.ReadFile(filepath.Join(rootPath, featuresSysPath, "policy", "permstable32")); err == nil {
+		permstableFeatures := strings.Split(string(data), " ")
```

Comment:
Instead of looking specifically for the prompt feature here, should we instead gather all of the features and return them all? The question then is how to represent them - perhaps as comma separated? Also I think these should be namespaced within the `policy:permstable32` prefix - so that we end up with something like: `policy:permstable32:allow,deny,subtree,cond,kill,complain,prompt,audit,quiet,hide,xindex,tag,label` - but perhaps @pedronis may have a better idea on how to represent this?

Reply:
Sure, this sounds good. I wasn't sure if things like `allow`, `deny` etc. would be considered "features" in the same sense, but I agree that exposing everything in the file makes sense.

If they're treated as similar to other features, I'd propose namespacing them like
```go
[]string{"policy:permstable32:allow", "policy:permstable32:deny", ..., "policy:permstable32:prompt", ...}
```
This would also allow checking for a particular feature without any parsing -- just, "does features list contain `"policy:permstable32:prompt"`?"

But if we treat `policy:permstable` as the "feature" with further details, I like your proposal of
```go
policy:permstable32:allow,deny,subtree,cond,kill,complain,prompt,audit,quiet,hide,xindex,tag,label
```
To check for prompt then, one would have to iterate through the features until finding the one starting with `"policy:permstable32:"`, and then either check if the complete feature contains `"prompt"` or actually split the remainder of the feature on commas and check for containment in that list.

Personally, I favor the former, since it's easier to parse while still remaining well-namespaced. The latter may be a bit more semantically correct, but not particularly consistent with other feature naming. Open to either, though.

Reply:
Yeah I was initially thinking the same (ie to split them as separate features) - so lets go with that instead so we can keep the parsing/testing for features easier.

Reply:
@alexmurray I pushed the change we discussed, whenever you get the chance to have another look